src/coreclr/vm/ecalllist.h

```diff
@@ -724,6 +729,7 @@ FCClassElement("Array", "System", gArrayFuncs)
 FCClassElement("AssemblyLoadContext", "System.Runtime.Loader", gAssemblyLoadContextFuncs)
 FCClassElement("Buffer", "System", gBufferFuncs)
 FCClassElement("CastHelpers", "System.Runtime.CompilerServices", gCastHelpers)
+FCClassElement("ComAwareWeakReference", "System", gComAwareWeakReferenceFuncs)
```

Comment:
I see that you've added the new FCalls but not removed the FCalls you've deleted from this file. Please do so and remove their implementations.

Reply:
Deleting unnecessary stuff is the "Undone" that I mentioned in the PR. 
I did not want to clatter code review with lots of edits that just delete code, while we are iterating on the design. I will do a dead code cleanup prior to merging this.

Thanks for noticing though. I will use this comment for tracking the removal.

Reply:
@davidwrighton I've removed a bunch of no longer reachable code. Including all the original WeakReference FCalls.